cmd/kubectl/kubectl.go

```diff
@@ -112,7 +112,8 @@ func NewK0sKubectlCmd() *cobra.Command {
 	originalRun := cmd.Run
 	cmd.RunE = func(cmd *cobra.Command, args []string) error {
 		if len(args) > 0 {
-			if err := kubectl.HandlePluginCommand(&kubectlPluginHandler{}, args); err != nil {
+			// FIXME need to think more about KEP-3638 here...
```

Comment:
After a quick look I don't see any obvious problem. The handler should handle it just like upstream does. Whether it should be `true` or `false` I'm not sure about. I assume the idea is that if you have `kubectl-create-foo` and `kubectl-create`, with "true" it will end up running `kubectl-create-foo` (exact match), with false it keeps searching for shorter matches and if it finds the `kubectl-create` it runs that instead.

Reply:
See #2994.